docs/PERMISSIONS.md

```diff
@@ -57,9 +57,9 @@ Mizu needs following permissions on your Kubernetes cluster to run properly
   - get
 ```
 
-## Permissions required for service / pod name resolving (opt)
+## Permissions required for service / pod name resolving
 
-Optionally, for proper resolving of IP addresses to Kubernetes service name, Mizu needs below permissions:
+Optionally (mandatory with `--daemon` flag), for proper resolving of IP addresses to Kubernetes service name, Mizu needs below permissions:
```

Comment:
A mandatory persions running in daemon mode, or optional permission needed for proper resolving of IP addresses to Kubernetes service name